ipcalc.c

```diff
@@ -599,7 +599,7 @@ char *ipv6_prefix_to_hosts(char *hosts, unsigned hosts_size, unsigned prefix)
 #define FLAGS_TO_IGNORE_MASK (~FLAGS_TO_IGNORE)
 
 static
-int get_ipv4_info(const char *ipStr, int prefix, ip_info_st * info,
```

Comment:
As it is never modified, I think having the ipStr marked as const is the right thing here.